org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/utils/ASTHelper.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copied from org.eclipse.jdt.internal.ui.util.ASTHelper
```

Comment:
@rgrunber any chance we could move that class to jdt-core so we can reuse it?

Reply:
From what I can see, this class is in : org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/util/ASTHelper.java , so it should be possible to use much like other things within jdt.core.manipulation (internal, or API)

Reply:
@snjeza can you please use that core.manipulation class instead?

Reply:
Done.